src/main/scala/Keys.scala

```diff
@@ -26,7 +26,7 @@ object Keys {
     "org.jetbrains.kotlin" % ("kotlin-" + name) % kotlinVersion.value
 
   def kotlinPlugin(name: String) = sbt.Keys.libraryDependencies +=
-    "org.jetbrains.kotlin" % ("kotlin-" + name) % kotlinVersion.value % "compile-internal"
+    "org.jetbrains.kotlin" % ("kotlin-" + name) % kotlinVersion.value % "compile-internal" exclude("org.jetbrains.kotlin", "kotlin-compiler-embeddable")
```

Comment:
I don't understand what the purpose of this is? Any transitive dependency will be under `compile-internal`